src/components/table/table.utils.test.js

```diff
@@ -0,0 +1,77 @@
+import {
+  generateTable,
+  generateTableCell,
+  generateTableRow,
+} from "./table.utils";
+import { COMPONENT_TYPES } from "./table.constants";
+
+describe("Table Utilities", () => {
+  describe("generateTableCell", () => {
```

Comment:
Create unit test cases to make sure the input arguments are defaulted and the output is as expected.